rocketmq-connect-jms/README-CN.md

```diff
@@ -0,0 +1,16 @@
+##### ActiveConnector完全限定名
+org.apache.rocketmq.connect.activemq.connector.ActivemqConnector
```

Comment:
is rocketmq-connect-jms folder wrong for this pr?

Reply:
Deleted ， ok